### PR TITLE
fix: links color, hover, and active states, etc

### PR DESCRIPTION
**Pull request by jsimplicio**
I messed up how links worked when I made the color changes, this should fix it.

### CODE REVIEW

File: public/account.html [hunk 1]
```diff
@@ -83,7 +83,9 @@
 
           <p>
             <button class="primary" type="submit">Send</button>
-            <a href="#" data-action="show-signin">Cancel</a>
+            <small>
```
Comment (nickcolley) [inline public/account.html]: Are the additions of '<small>' related? Just thinking about touch area sizes

Comment (jsimplicio) [inline public/account.html]: Additions of what? Sorry didn't understand :sweat_smile:

**Comment by jsimplicio** [comment]
Before:
screen shot 2016-03-15 at 6 32 35 pm
After:
screen shot 2016-03-15 at 6 33 01 pm

File: public/account.html (hunk 1 above)
Comment (jsimplicio) [inline public/account.html]: Was able to understand your comment. I forgot to apply `<small>` to these things before but every action next to a button has the `<small>` applied to it, it makes it look more visually balanced that the secondary action font size isn't as big as the primary button to the left of it.

But if this causes any problem I can remove it!

Comment (jsimplicio) [inline public/account.html]: @nickcolley  Gregor offered a suggestion to make a `.btn-link` class that is applied to these choices that are buttons that just look like a link. In the class we can add enough padding just like the buttons have or similar. How's that fix?

Comment (gr2m) [inline public/account.html]: we discussed it with Julia. Julia suggested to make an extra class for links like `cancel`, similar to Bootstrap’s `btn-link` class

Comment (jsimplicio) [inline public/account.html]: I did it and pushed it here if you wanna look.

**Comment by gr2m** [comment]
I think with the `.btn-link` class it’s great! LGTM

**Comment by nickcolley** [comment]
I was thinking of doing a patch with more padding for links so LGTM :+1:

**Comment by nickcolley** [comment]
Thanks @jsimplicio :)

**Pull request merged**